website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -0,0 +1,683 @@
+---
+id: research-reconciliation
+title: ORCA Reconciliation
+description: Initial notes, design and ideas related to ORCA Reconciliation with Cumulus
+---
+import MyImage from '@site/docs/templates/pan-zoom-image.mdx';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+As part of enhancing ORCA, a needed feature by the working group is being able to
+reconcile between the ORCA inventory and the Cumulus inventory to determine what
+file(s) are missing or extra in each catalog. The ORCA team has met with Cumulus
+several times to understand current reconciliation patterns. Notes from those
+interface meetings are available below. In addition, there are several resources
+available from Cumulus and LZARDS which also has to deal with reconciliation in
+a manor similar to ORCA. It is understood that ORCA will provide the data but,
```

Comment:
<!--StartFragment-->a manor similar to ORCA. It is understood that ORCA will provide the data, but Cumulus will perform the comparison and reporting work to match current reconcile<!--EndFragment-->

Reply:
fixed.